layouts/shortcodes/working-groups.html

```diff
@@ -19,7 +19,11 @@ <h2 id="{{ .name | urlize }}">{{ .name | title }}</h2>
     <div class="container margin-top-30 margin-bottom-30">
       <div class="row text-center">
         {{ range .chairs }}
+        {{if eq .name "Jérôme Quevremont"}}
```

Comment:
we shouldn't be hardcoding on a name. We should check if the length of chairs is 1 `if eq (len $chairs) 1`. We'd need to grab a copy of the chairs before hand though

Reply:
Oh, yeah! Checking the length would be much better!